client/removinator/removinator.py

```diff
@@ -159,6 +164,48 @@ def remove_card(self):
 
         self.send_command('REM')
 
+    def lock_card(self, slot):
+        """
+        Lock a specified card slot
+
+        This sends the LOCK command to the controller, with 'slot' as the
+        smart card slot number.
+
+        :param slot: the slot of the card to insert (1-8)
+        :type slot: int
+        :raises: :exc:`SlotError`, :exc:`CommandError`, :exc:`ValueError`
+        """
+
+        if slot < 0 or slot > 8:
+            raise ValueError('Slot must be between 1 and 8')
+
+        try:
+            self.send_command('LOCK{0}'.format(slot))
+        except CommandError as e:
+            if self.last_result == 'ERR_NOCARD':
+                raise SlotError('No card present in slot {0}'.format(slot)) from None
+            elif self.last_result == 'ERR_LOCKED':
+                raise LockedError('Slot {0} is already locked'.format(slot)) from None
+            else:
+                raise e from None
+
+    def unlock_card(self, slot):
+        """
+        Unlock a specified card slot
+
+        This sends the UNLOCK command to the controller, with 'slot' as the
+        smart card slot number.
+
+        :param slot: the slot of the card to insert (1-8)
+        :type slot: int
+        :raises: :exc:`SlotError`, :exc:`CommandError`, :exc:`ValueError`
+        """
+
+        if slot < 0 or slot > 8:
+            raise ValueError('Slot must be between 1 and 8')
+
+        self.send_command('UNLOCK{0}'.format(slot))
+
     def get_status(self):
```

Comment:
It would be good to also  update the docstring for get_status() to mention that a "locked" key will now be returned  to indicate which card slots are locked.

Reply:
In general, your changes look really good!  It would be nice to update the "Firmware" section of the main README to briefly describe the new lock/unlock commands.

Reply:
I will probably reword it to point to the source code. This project already suffers from many copies of the same information (in firmware, in python code, in docstrings, in Doxygen, in documentation, etc.).